DQMOffline/Trigger/python/MuonMonitor_cff.py

```diff
@@ -0,0 +1,34 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.MuonMonitor_cfi import hltMuonmonitoring
+
+
+TrkMu12_DoubleTrkMu5NoFiltersNoVtx_monitoring = hltMuonmonitoring.clone()
+TrkMu12_DoubleTrkMu5NoFiltersNoVtx_monitoring.FolderName = cms.string('HLT/Exotica/TrkMu12_DoubleTrkMu5NoFiltersNoVtx/')
+TrkMu12_DoubleTrkMu5NoFiltersNoVtx_monitoring.numGenericTriggerEventPSet.hltPaths = cms.vstring("HLT_TrkMu12_DoubleTrkMu5NoFiltersNoVtx_v*")
```

Comment:
I do not see this path, but 
HLT_TrkMu15_DoubleTrkMu5NoFiltersNoVtx_v* seems to not be covered, instead